redgenes/gene_annotations.py

```diff
@@ -1,7 +1,11 @@
+import sys
+import time
+import numpy as np
```

Comment:
```suggestion
import numpy as np
import click
import atexit
```

...relevant for suggested changes below